### PR TITLE
Text bug fixes

### PR DESCRIPTION
**Pull request by vitalii-bulyzhyn**
## Description
...

### Author Checklist
_All items are required. Please add a note to the item if the item is not applicable and please add links to any relevant follow up issues._

I have...

* [ ]  added `!` to the type prefix if Breaking Changes.
* [ ]  considered the impact to the SDK.
* [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
* [ ]  provided a link to the relevant issue in JIRA or Github Link in PR.
* [ ]  included the unit and integration tests
* [ ]  included comments & docs for new API Endpoints
* [ ]  updated the relevant documentation or specification in readme.io
* [ ]  reviewed "Files changed" and left comments if necessary
* [ ]  confirmed all CI checks have passed

Breaking Changes including adding required params.

### Reviewers Checklist
_All items are required. Please add a note if the item is not applicable and please add your handle next to the items reviewed if you only reviewed selected items._

I have...

* [ ]  confirmed the Author checklist was followed
* [ ]  reviewed API design and naming
* [ ]  reviewed documentation is accurate
* [ ]  reviewed tests and test coverage
* [ ]  manually tested (if applicable)

### CODE REVIEW

**Comment by vitalii-bulyzhyn** [comment]
Way to reproduce freezing bug:

Screenshot 2022-11-23 at 17 28 48

File: frontend/src/components/text_annotation/text_annotation_core.vue [hunk 1]
```diff
@@ -70,15 +70,18 @@
         :context_menu="context_menu"
         @delete_instance="delete_instance"
       />
+      <div v-if="resizing">Resizing...</div>
+      <div v-if="rendering">Loading...</div>
```
Comment (anthony-sarkis) [inline frontend/src/components/text_annotation/text_annotation_core.vue:74]: @vitalii-bulyzhyn  want to add v-linear loader here? like from image case?

**Review by PJEstrada — CHANGES_REQUESTED** [review]
Hey Vitalii! this is the only  thing I got:

1. When displaying the labels menu sometimes it gets hidden by the toolbar
image
I think the fix might be to add a scrollbar to the labels menu 

For the rest of issues I think they are all fixed now Vitalii thanks a lot :)

**Comment by anthony-sarkis** [comment]
Great work Vitalii, I tested most of these again on QA and all worked as expected.

**Review by anthony-sarkis — APPROVED** [review]

**Review by PJEstrada — APPROVED** [review]
Thanks Vitalii!

**Pull request merged**